app/views/maintainers/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<th class="container">
+  <h2>Add Maintainer for <strong><%= @project.title %></strong></h2>
+  <table class="table table-striped">
+    <thead><tr>
+      <th scope="col">#</th>
+      <th scope="col">Name</th>
+      <th scope="col">Add or Remove</th>
+    </tr></thead>
+    <tbody>
+      <% User.all.each do |user| %>
```

Comment:
Ez egy nogo. View-ban nem csinálunk query-ket. Helyette legyen a kontrollerakcióban egy @users vagy @maintainers változó, amiben le-query-zzük a user-eket.

Reply:
Ezen kívül innen nagyon hiányzik egy policy_scope hívás. Melyik user-eket láthatjuk egy adott projekt alatt? Van bármi speckó szabály?

Reply:
Itt mindegyik usert látod, mert itt állítod be, hogy ki legye na projekten. Annak megfelelően jelenik meg "Add" vagy "Remove", hogy rajta van e a projekten már vagy sem.

Reply:
Erről beszéltünk meeting-en. `policy_scope` -ot érdemes lesz használnunk, mert így marad könnyen bővíthető az alkalmazás. Érdemes megvizsgálnunk, hogy melyik user-eket listázhatja ki az aktuális felhasználó, mielőtt mindet odaadjuk neki.

Jelenleg a `UserPolicy` `resolve` -ja `scope.all` -lal tér vissza, így a jelenlegi működésen nem fog változtatni.
(Ha változtatna, akkor a `UserPolicy` -t lenne érdemes átalakítani.)